test/v2/controllers/requestParsers/validators/validations/HistoricTaxPeriodYearValidationSpec.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.controllers.requestParsers.validators.validations
+
+import support.UnitSpec
+import v2.models.errors.{ RuleHistoricTaxYearNotSupportedError, TaxYearFormatError }
+
+class HistoricTaxPeriodYearValidationSpec extends UnitSpec {
+
+  val minTaxYear = 2017
+  val maxTaxYear = 2021
+  val taxDate    = "2020-08-02"
+
+  "validate" should {
+    "return no errors" when {
+      "a valid taxDate is supplied" in {
+        val validationResult = HistoricTaxPeriodYearValidation.validate(minTaxYear, maxTaxYear, taxDate)
```

Comment:
You could probably just call these "result" as we already know it's a ValidationSpec. Also less verbose :)